eo-phi-normalizer/CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the
 [Haskell Package Versioning Policy](https://pvp.haskell.org/).
 
+## v0.4.1 — 2024-06-12
```

Comment:
Summary hides some of the important changes behind links to PRs. Take the specific changes from PR descriptions and put them here. Also, please, check that all PRs are reflected in the summary.

Reply:
@fizruk, extended